### PR TITLE
add PartialEq impl

### PR DESCRIPTION
**Pull request by joseildofilho**
Hi There, 
I'm using your lib to a university project, so a feel that was a good idea to pull my changes, hope you like.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -31,6 +32,15 @@ pub struct BitLongVec {
     pub data: Vec<u64>,
 }
 
+impl PartialEq for BitLongVec {
```
Comment (vaIgarashi) [inline src/lib.rs]: Why do we need manual trait implementation? We can use `#[derive(PartialEq)]`.

File: src/lib.rs [hunk 2]
```diff
@@ -138,6 +148,30 @@ impl BitLongVec {
...
+#[test]
+#[should_panic]
+fn test_partial_eq_data_diff_attr() {
+    let mut vec = BitLongVec::with_fixed_capacity(1, 63);
+    let mut vec2 = BitLongVec::with_fixed_capacity(63, 1);    
+    vec.set(0, 2u64.pow(63)-1);
+    for i in 0..63 {
+        vec2.set(i, 1);
+    }
+    assert_eq!(vec, vec2);
```
Comment (vaIgarashi) [inline src/lib.rs]: It would be better to expect correct condition withouth panic.

```rust
assert!(vec != vec2);
```

**Review by vaIgarashi — CHANGES_REQUESTED** [review]
Hello! Thanks for submiting your pull request. I have added some comments. It's possible to implement `PartialEq` automaticly using derive:  `#[derive(Debug, PartialEq, Eq)]`. You can also remove tests because it's not neccesery to test rust internals

File: src/lib.rs (hunk 2 above)
Comment (vaIgarashi) [reply src/lib.rs]: or

 ```rust
assert_ne!(vec, vec2);
```

**Comment by joseildofilho** [comment]
I belive that now it's in a movre "Rustic" way ? 
P.S.: I'm new in rust, somethings arent totally clear, but i really appreciate your suggesttion

**Review by vaIgarashi — APPROVED** [review]

**Pull request merged**